bedrock/firefox/views.py

```diff
@@ -687,7 +687,9 @@ def get_template_names(self):
         if variant not in self.variations:
             variant = None
 
-        if ftl_file_is_active('firefox/new/download'):
+        if variant == 'a':
```

Comment:
not necessarily, because a convert experiment on /new will link to this page & the cloudflare worker will handle locale. Going to add in anyway just to be explicit.